### ISSUE TITLE
Consider joining official Ansible vscode extension

### ISSUE THREAD

**Issue by ssbarnea** [rupisaini/vscode-ansible-linter#13]
Hi @rupisaini! I am the current maintainer of the ansible-lint and I was pleased that I was not the only one that felt the important of getting its results integrated into vscode.

Back in december I started a project that aims to become the [official ultimate extension for Ansible](https://marketplace.visualstudio.com/items?itemName=zbr.vscode-ansible) and its first usable feature is support for ansible-lint.

Please give it a go as this one should really work with current versions of vscode and ansible-lint and see this as my invitation to join efforts on maintaining a vscode extension. I personally thing is much better to have less extensions but better maintained, especially as I want its development to be fully driven by the community.

**Comment by ssbarnea** [comment]
@rupisaini In case you no longer have time or interest on maintaining your ansible-lint extensions, would you mind unpublishing it from the marketplace? I am asking this because presence of broken/outdated/unmaintained extensions dillute the marketplace usability, making harder to find active ones.

**Comment by rupisaini** [comment]
Hi Sorin,

Sorry for late reply. I haven't worked on Ansible for couple of years now.
If you have another working Ansible lint extension then I can unpublish  my
or you can take over this (if possible).

**Comment by ssbarnea** [comment]
Yes, i recently published https://marketplace.visualstudio.com/items?itemName=zbr.vscode-ansible which adds support for ansible-lint and I plan to maintain it. That extension does not have ansible-lint in its name because it aims to cover multiple tools around Ansible, not only the linter. 

If you could go to the marketplace and unpublish it, so other users will not find it when searching for "ansible" it would be awesome.  If you want, you could also change the github repository description with something like: RETIRED in favour of https://github.com/ansible-community/vscode-ansible -- it will be awesome, as I am looking for others to join the development.
